osu.Game.Tests/Visual/TestCaseMods.cs

```diff
@@ -182,13 +182,16 @@ private void testMultiplierTextColour(Mod mod, Color4 colour)
             checkLabelColor(Color4.White);
         }
 
-        private void testMultiplierTextUnranked(Mod mod)
+        private void testRankedText(Mod mod)
         {
-            AddAssert("check for ranked", () => !modSelect.MultiplierLabel.Text.EndsWith(unranked_suffix));
+            AddWaitStep(1, "wait for fade");
+            AddAssert("check for ranked", () => modSelect.RankedLabel.Alpha.Equals(0));

```

Comment:
Why didn't you use "==" 🤔

Reply:
Is there a difference, besides obvious syntax?